data/assets/scene/solarsystem/heliosphere/bastille_day/fluxnodescutplane.asset

```diff
@@ -0,0 +1,113 @@
+local assetHelper = asset.require("util/asset_helper")
+local propertyHelper = asset.require('../../../../util/property_helper')
```

Comment:
I think this should just be `"util/property_helper"`, similarly to the asset helper above